### DISCUSSION TITLE
Hive dialect?

### DISCUSSION THREAD

**Discussion by eitsupi** [PRQL/prql#2831] [General]
Now that the ClickHouse integration test is working, the only dialect currently implemented that has not been tested is Hive, except for those that are not substantive or not OSS.

https://github.com/PRQL/prql/blob/5425a67e16e1dcb301f98bf58b3aa2af3085f561/prql-compiler/src/sql/dialect.rs#L44-L57

So I would like to check if we can add a test for Hive, but what do you mean by Hive dialects here?
I looked at the PR #347 which add Hive dialect and couldn't figure out why Hive is added.

I'm also wondering if it's worth keeping the Hive dialect, since I'm not sure if it's worth it to have the Hive dialect in the first place.
(I have used Hive and Spark very little!)

**Comment by max-sixty** [comment]
I'm fine removing it. I can't remember why we had it, and it's better to have nothing rather than something that doesn't work. 

We can always wait for someone to come along and champion it...

**Reply by max-sixty** [reply]
(SparkSQL might be more useful than Hive, but also we can wait for a champion)

**Reply by eitsupi** [reply]
Thank you, perhaps it would be better to remove it so as not to cause confusion.
